hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/EventCondition.java

```diff
@@ -84,9 +99,12 @@ public void setExpression(String expression) {
     }
 
     public boolean match(Event value) {
-        if (null == expression || expression.isEmpty() || null == value) {
+        if (null == value) {
             return false;
         }
+        if (null == expression || expression.isEmpty()) {
+            return true;
+        }
         String[] expressions = expression.split(",");
```

Comment:
event.id start 'ID,YZ', event.context.category == 'Ser,ver', event.tag.from end ',com'
will not be parsed correctly, as the split will break in incorrect tokens.
I'll update this with a better parsing algorithm covering escaped commas and quote inside the string argument.